content/shmem_malloc.tex

```diff
@@ -54,7 +54,11 @@
     whether an existing allocation is modified and whether new memory is allocated.
     This ensures that all
     \ac{PE}s participate in the memory allocation, and that the memory on other
-    \ac{PE}s can be used as  soon as the local \ac{PE} returns.  The user is
+    \ac{PE}s can be used as  soon as the local \ac{PE} returns.
+    \newtext{The user is responsible for ensuring that no communication
```

Comment:
That's the intended semantic; however, I think you're right that the text is too restrictive.  I just pushed a fix.  Could you take a look at the new text?